psync/src/main/java/net/named_data/jni/psync/PSync.java

```diff
@@ -79,8 +79,8 @@ public void removeUserNode(String prefix) {
             removeUserNode(m_buffer, prefix);
         }
 
-        public long getSeqNo(String prefix) {
-            return getSeqNo(m_buffer, prefix);
+        public void getSeqNo(String prefix) {
```

Comment:
Why is this changed to void? Someone might have use for seqNo even if we don't.

Reply:
Oops. I initially copied the files from the JNI PSync github and it's void there. Must have been something I fixed here but never changed there. After we merge this, I'll create a pull request there.